CLAUDE.md

```
@@ -829,59 +829,509 @@ Response
+
+3. **Filter Excluded Assets**
+   ```java
+   candidates = filterExcludedAssets(candidates, deckAssetIds + shownAssetIds);
+   ```
+
+### Selection Process
+
+#### 1) Normal Picks (2개)
+선호 섹터 + 리스크 프로필 + 다양성 패널티 반영
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)